couchdb/couchdb.rb

```diff
@@ -81,7 +81,7 @@ def parent_interrupt
   ['--apihost',   '-a', GetoptLong::REQUIRED_ARGUMENT]
 )
 
-config_file = "config.yml"
+config_file = "/usr/local/ucm/couchdb/config.yml"
```

Comment:
setup a base path variable, and then use that to relatively call out different config files